playbooks/roles/cyclecloud_cluster/projects/common/cluster-init/scripts/90-mktmp.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# written by Jeff Jones. Slight modifications by Matt Chan
+
+if [ ! -e /etc/redhat-release]; then
```

Comment:
this should be removed to cover other distros. In 5-default.sh.j2 we make sure /mnt/resource is created. Covering other distros means that the NHC test will succeed

Reply:
Ah. Yes that's a good point!